Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -81,6 +89,11 @@ public override IEnumerable<ISetting> GetSettings()
             _urlSettings.CustomControl = new TextBox();
             yield return _urlSettings;
 
+            _authTypeSettings.CustomControl = new ComboBox { DropDownStyle = ComboBoxStyle.DropDownList };
+            _authTypeSettings.CustomControl.Items.AddRange(_authTypeSettings.Values.ToArray());
```

Comment:
[nit] This creates unnecessary allocations. Suggest unrolling this into `foreach (... value in _authTypeSettings.Values) { _authTypeSettings.CustomControl.Items.Add(item); }`

Reply:
got it